cpp/tests/copying/utility_tests.cu

```diff
@@ -87,7 +87,7 @@ void check_empty_string_columns(cudf::column_view lhs, cudf::column_view rhs)
     EXPECT_EQ(lhs.null_count(), 0);
     EXPECT_EQ(lhs.nullable(), false);
     EXPECT_EQ(lhs.has_nulls(), false);
-    EXPECT_EQ(lhs.num_children(), rhs.num_children());
+    // An empty column is not required to have children
```

Comment:
Should the same change be made in `expect_column_properties_equal` in  cpp/tests/utilities/column_utilities.cu?

Reply:
`expect_columns_equal` should check for _exact_ equality. I can't recall who, but someone is adding `expect_columns_equivalent` that checks for semantic equivalence (which should do the check you suggest).